CustomTabBar/CustomTabBar.swift

```diff
@@ -69,12 +69,24 @@ public protocol CustomTabBarDelegate {
         }
     }
     
+    public var titleFont: UIFont = UIFont.systemFontOfSize(16) {
```

Comment:
I think I'm scared about this. We worked hard to get our tab names to live inside the bounds, and 16 probably won't do it. I thought we landed at 10 or something, but I don't see that line in this diff.